src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,18 @@
+import Card from "react-bootstrap/Card";
+
+export const ArticleCard = ({ title, imgUrl, author, createdAt }) => {
+  return (
+    <div className={"article-card"}>
+      <Card style={{ width: "18rem" }}>
+        <Card.Img variant="top" src={imgUrl} />
+        <Card.Body>
+          <Card.Title>{title}</Card.Title>
+          <Card.Text>
+            Author: {author} <br />
+            Date: {createdAt}
```

Comment:
Making these properties a bit less developer-y would be worthwhile. especially the way the date is displayed